### PR TITLE
List all practice commands when enabling `/practice`

### PR DESCRIPTION
**Pull request by furo321**
image

## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/engine/shared/config.h [hunk 1]
```diff
@@ -50,12 +50,13 @@ enum
 	// DDRace
 
 	CMDFLAG_TEST = 1 << 6,
-	CFGFLAG_CHAT = 1 << 7,
-	CFGFLAG_GAME = 1 << 8,
-	CFGFLAG_NONTEEHISTORIC = 1 << 9,
-	CFGFLAG_COLLIGHT = 1 << 10,
-	CFGFLAG_COLALPHA = 1 << 11,
-	CFGFLAG_INSENSITIVE = 1 << 12,
+	CMDFLAG_PRACTICE = 1 << 7,
```
Comment (heinrich5991) [inline src/engine/shared/config.h:53]: The flags go into teehistorian, could you add it at the end instead?

File: src/game/server/ddracechat.cpp [hunk 2]
```diff
@@ -668,13 +668,26 @@ void CGameContext::ConPractice(IConsole::IResult *pResult, void *pUserData)
...
-	str_format(aBuf, sizeof(aBuf), "'%s' voted to %s /practice mode for your team, which means you can use /r, but you can't earn a rank. Type /practice to vote (%d/%d required votes)", pSelf->Server()->ClientName(pResult->m_ClientId), VotedForPractice ? "enable" : "disable", NumCurrentVotes, NumRequiredVotes);
+	str_format(aBuf, sizeof(aBuf), "'%s' voted to %s /practice mode for your team, which means you can use practice commands, but you can't earn a rank. Type /practice to vote (%d/%d required votes)", pSelf->Server()->ClientName(pResult->m_ClientId), VotedForPractice ? "enable" : "disable", NumCurrentVotes, NumRequiredVotes);
 	pSelf->SendChatTeam(Team, aBuf);
 
 	if(NumCurrentVotes >= NumRequiredVotes)
 	{
 		Teams.SetPractice(Team, true);
 		pSelf->SendChatTeam(Team, "Practice mode enabled for your team, happy practicing!");
+
+		char aPracticeCommands[1024];
```
Comment (heinrich5991) [inline src/game/server/ddracechat.cpp:679]: 1024 is probably too long for a single chat message. Was the limit 256?

**Comment by furo321** [comment]
Thanks, fixed!

**Review by heinrich5991 — APPROVED** [review]
Thanks!

**Pull request merged**